app/services/client.py

```diff
@@ -0,0 +1,22 @@
+import requests
+from dotenv import load_dotenv
+from os import getenv
+
+
+load_dotenv()
+
+class WordsClient:
+    def __init__(self, base_url: str, apiKey: str) -> None:
+        self.base_url = base_url
+        self.apiKey = apiKey
+
+
+    def RandomWord(self):
```

Comment:
this is a regular function even if its inside the class, so it should be with snake_case